src/H5public.h

```diff
@@ -94,10 +94,10 @@ extern "C" {
 /* Version numbers */
 #define H5_VERS_MAJOR      1                               /* For major interface/format changes           */
 #define H5_VERS_MINOR      8                               /* For minor interface/format changes           */
-#define H5_VERS_RELEASE    22                              /* For tweaks, bug-fixes, or development     */
-#define H5_VERS_SUBRELEASE "snap12"                        /* For pre-releases like snap0       */
+#define H5_VERS_RELEASE    23                              /* For tweaks, bug-fixes, or development     */
+#define H5_VERS_SUBRELEASE "0"                        /* For pre-releases like snap0       */
```

Comment:
I believe this is the issue with '0', because H5_VERS_SUBRELEASE is evaluated as nothing.